tests/tests/tier0/bluechi-agent-set-loglevel/test_bluechi_agent_set_loglevel.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+
+NODE_FOO = "node-foo"
+
+
+def get_str_to_set_loglevel(node, loglevel):
+    return f"bluechictl set-loglevel {node} {loglevel}"
+
+
+def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+    node_foo = nodes[NODE_FOO]
+
+    loglevel = "DEBUG"
+    ctrl.exec_run(get_str_to_set_loglevel(NODE_FOO, loglevel))
+    ctrl.exec_run("bluechictl metrics enable")
+    ctrl.exec_run("bluechictl metrics disable")
+
+    loglevel = "INFO"
+    ctrl.exec_run(get_str_to_set_loglevel(NODE_FOO, loglevel))
+    ctrl.exec_run("bluechictl metrics enable")
+    ctrl.exec_run("bluechictl metrics disable")
+    _, output = node_foo.exec_run("journalctl -u bluechi-agent.service")
+
+    sub_output = output.split("Log level changed to DEBUG")
+    assert len(sub_output) == 2
+    sub_output = sub_output[1].split("Metrics enabled")
+    assert len(sub_output) == 2
+    sub_output = sub_output[1].split("Metrics disabled")
+    assert len(sub_output) == 2
```

Comment:
Why do we enable/disable metrics, when there is no difference in logging between INFO and DEBUG log levels?